plugins/extractors/googleworkspace/googleworkspace.go

```diff
@@ -0,0 +1,219 @@
+package googleworkspace
+
+import (
+	"context"
+	_ "embed" // used to print the embedded assets
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/odpf/meteor/models"
+	"github.com/odpf/meteor/plugins"
+	"github.com/odpf/meteor/registry"
+	"github.com/odpf/meteor/utils"
+	"github.com/odpf/salt/log"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	admin "google.golang.org/api/admin/directory/v1"
+	"google.golang.org/api/option"
+
+	commonv1beta1 "github.com/odpf/meteor/models/odpf/assets/common/v1beta1"
```

Comment:
@ishanarya0 Since meteor models have been updated, can you use v1beta2 for models here. Feel free to check other extractors as references.